pkgs/development/coq-modules/serapi/default.nix

```diff
@@ -0,0 +1,72 @@
+{ lib, fetchzip, mkCoqDerivation, coq, version ? null }:
+
+let
+  ocamlPackages =
+    coq.ocamlPackages.overrideScope'
+      (self: super: {
+        ppxlib = super.ppxlib.override { version = "0.15.0"; };
```

Comment:
Is it necessary to override this set-wide? Inlining this below would probably speed up eval as you don't need to recompute the `ocamlPackages` fix point.

Reply:
@sternenseemann If you don't mind, could you push a commit with your suggested changes? It might take me a while to figure out how to apply them by myself. Thanks!